src/js/components/ScaleList.jsx

```diff
@@ -1,8 +1,22 @@
 'use strict';
 var React = require('react'),
+    connect = require('react-redux').connect,
     ContentEditable = require('./ContentEditable');
 
-var ScaleList = React.createClass({
+function mapStateToProps(reduxState, ownProps) {
+  return {
+    selected: reduxState.get('selectedMark')
+  };
+}
+
+var ScaleList = connect(
```

Comment:
I take it back, we can't actually make this change right now without further modularizing the code or making otherwise-unnecessary intermediate identifiers (LayerList for example has a non-exported, `connect`ed `Group` component that is used within the exported class). We _should_ fragment this, and I am noting this as a preferred code style, but I would like to keep this as-is for the purpose of this PR.